projectgenerator/libqgsprojectgen/generator/postgres/domain_relations.py

```diff
@@ -129,7 +131,7 @@ def find_domain_relations(self, layers, conn, schema, inheritance):
                             if self.debug: print("Relation:", relation.name)
                             relations.append(relation)
 
-        if self.debug: print("final_models_info:", models_info)
+        if self.debug: print("final_models_info:", models_info_with_ext)
```

Comment:
Just an unrelated comment, I think we should switch all these `print` statements to the `logging` module, so we can send it to stdout, files, message log ...

And who knows, a `QgsMessageLogHandler` or similar for `logging` might even be a candidate for shipping directly with QGIS 3.